modopt/opt/algorithms.py

```diff
@@ -148,7 +132,10 @@ def metrics(self, metrics):
 
         if isinstance(metrics, type(None)):
             self._metrics = {}
-        elif not isinstance(metrics, dict):
+            return
```

Comment:
What's the point of this `return` statement?

Reply:
Old code, removing